rrrspec-client/lib/rrrspec/client/cli.rb

```diff
@@ -89,6 +98,7 @@ def waitfor(taskset_id)
           elsif rs.size != 0
             rs[0].getc
             taskset.cancel
+            break
```

Comment:
I have a doubt on this line. With this break, the process will quit without waiting for the taskset.